locales/kin.json

```diff
@@ -64,5 +64,11 @@
   "Line manager is required": "Izina rya manager rigomba kuzuzwa",
   "Profile is already complete": "Umwirondoro wawe uruzuye",
   "Profile completed succesfuly": "umwirondoro wujujwe neza",
-  "Profile updated succesfuly": "guhindura umwirondoro byagenze neza"
+  "Profile updated succesfuly": "guhindura umwirondoro byagenze neza",
+  "Reset password - Barefoot Nomad": "Guhindura umubare w'ibanga - Barefoot Nomad",
+	"Failed in sending Reset password email !!!": "Guhindura umubare w'ibanga ntibyakunze !!!",
+	"A link to reset your password has been sent to your email address !!!": "Ilinki yo guhindura umubare w'ibanga yoherejwe kuri imeri yanyu !!!",
+	"Email does NOT exist in our database !!!": "Imeri ntabwo iri muri sisitemu !!!",
+	"The passwords do NOT match !!!": "Imibare y'ibanga ntabwo isa !!!",
+	"Password well reset": "Guhindura umubare w'ibanga byagenze neza"
```

Comment:
the spacing on line 67-68 is different from 69-73

Reply:
I fixed that.